src/migmose/mig/reducednestednachrichtenstruktur.py

```diff
@@ -149,3 +233,16 @@ def to_json(self, message_type: EdifactFormat, output_dir: Path) -> dict[str, An
             json.dump(structured_json, json_file, indent=4)
         logger.info("Wrote reduced nested Nachrichtenstruktur for {} to {}", message_type, file_path)
         return structured_json
+
+    def to_tree(self, message_type: EdifactFormat, output_dir: Path) -> None:
+        """Writes reduced NestedNachrichtenstruktur in the .tree grammar of MAUS."""
+        # generate tree dict
+        tree_dict: DefaultDict[str, list[NachrichtenstrukturZeile]] = defaultdict(list)
+        tree_dict = _build_tree_dict(self, tree_dict)
+        # write tree file
+        output_dir.mkdir(parents=True, exist_ok=True)
+        file_path = output_dir.joinpath(f"{message_type}.tree")
```

Comment:
könntest du in den filename noch die mig-version reinschreiben? wie hier?
https://github.com/Hochfrequenz/edifact-templates/blob/main/edi/IFTSTA/IFTSTA2.0e.tree